x/escrow/handler.go

```diff
@@ -431,3 +439,13 @@ func loadEscrow(bucket Bucket, db weave.KVStore, escrowID []byte) (*Escrow, erro
 	}
 	return escrow, nil
 }
+
+func moveCoins(db weave.KVStore, bank cash.CoinMover, src, dest weave.Address, amounts []*coin.Coin) error {
```

Comment:
lgtm. very simple function, but used many places.
can be refactored to other location in the future, but good for now